public/app/api/createBaseQuery.ts

```diff
@@ -9,7 +9,7 @@ interface RequestOptions extends BackendSrvRequest {
   body?: BackendSrvRequest['data'];
```

Comment:
`showErrorAlert` is already in `BackendSrvRequest` so doesn't need redefining here